### PR TITLE
[wasm-bindgen]: make uid and block_number in decrypt_block function

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ All notable changes to this project will be documented in this file.
 
 
 ---
-## [0.4.1] - 2022-02-21
+## [0.5.0] - 2022-03-09
+### Added
+### Changed
+- [wasm-bindgen]: make uid and block_number in decrypt_block function
```
Comment (ThibsG) [inline CHANGELOG.md:10]: "make optional" you mean?

File: src/interfaces/wasm_bindgen/hybrid_gpsw_aes.rs [hunk 2]
```diff
@@ -121,26 +121,17 @@ pub fn webassembly_decrypt_hybrid_block(
...
-    log(&format!("webassembly: block_number {:?}", block_number));
-    log(&format!(
-        "webassembly: encrypted_bytes: {:?}",
-        hex::encode(encrypted_bytes.to_vec())
-    ));
-
+    let uid = match uid_bytes {
+        Some(uid) => uid.to_vec(),
+        None => vec![],
+    };
```
Comment (ThibsG) [inline src/interfaces/wasm_bindgen/hybrid_gpsw_aes.rs:127]: Use "map_or" here

File: CHANGELOG.md (hunk 1 above)
Comment (Manuthor) [reply CHANGELOG.md:10]: yes!

File: src/interfaces/wasm_bindgen/hybrid_gpsw_aes.rs (hunk 2 above)
Comment (Manuthor) [reply src/interfaces/wasm_bindgen/hybrid_gpsw_aes.rs]: done

**Pull request merged**